lib/ui/recovery_key_page.dart

```diff
@@ -29,7 +29,8 @@ class RecoveryKeyPage extends StatefulWidget {
       this.isDismissible,
       this.title,
       this.text,
-      this.subText})
+      this.subText,
+      this.showProgressBar = false})
```

Comment:
It is set to false by default everywhere, so only when the optional parameter showProgressBar is set to true when RecoveryKeyPage() is called will the progress bar be shown.